pywbem/_cim_operations.py

```diff
@@ -489,7 +497,7 @@ def __init__(self, url, creds=None, default_namespace=None,
         Parameters:
 
           url (:term:`string`):
-            URL of the WBEM server, in the format:
+            URL of the WBEM server or WBEM listener, in the format:
```

Comment:
Should we be saying something about the port being a requirement for the WBEMListener?

Reply:
Yes. Added a sentence.
DONE